neo/SmartContract/InteropService.cs

```diff
@@ -159,11 +159,11 @@ internal static bool CheckWitness(ApplicationEngine engine, UInt160 hash)
                                 // verify if context is correct for execution
                                 return engine.CurrentScriptHash == new UInt160(usage.Scope.ScopeData);
                             }
-                        case WitnessScopeType.RootAccess:
+                        case WitnessScopeType.EntryOnly:
                             {
                                 // verify if context is correct for execution
                                 return engine.CallingScriptHash == engine.EntryScriptHash;
-                            }
+                            }   
```

Comment:
spaces :)

Reply:
Very good.. I tried to format here, but my formatter only format the whole project, not separate files...
I'll need to fix whole formatting on other PR, then come back to this.